core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -130,6 +130,8 @@ class KafkaServer(val config: KafkaConfig, time: Time = Time.SYSTEM, threadNameP
   private var shutdownLatch = new CountDownLatch(1)
 
   private val jmxPrefix: String = "kafka.server"
+  private val KAFKA_CLUSTER_ID: String = "kafka.cluster.id"
+  private val KAFKA_BROKER_ID: String = "kafka.broker.id"
```

Comment:
we should add a comment to clarify those constants are all metrics context related.